### PR TITLE
new-news/liferea: version bump

### PR DESCRIPTION
**Pull request by nE0sIghT**
https://github.com/lwindolf/liferea/releases/tag/v1.12.2

### CODE REVIEW

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit autotools gnome2 pax-utils python-single-r1
+
+MY_P=${P/_/-}
+MY_PV=${PV/_/-}
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: This variable is used only once, please inline it.

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+S=${WORKDIR}/${MY_P}
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: You've got S declared twice. Please remove this one.

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+S="${WORKDIR}"/${MY_P}
+
+src_prepare() {
+	gnome2_src_prepare
+
+	# gnome2_src_prepare calls elibtoolize
+	if [ -f "${S}"/.elibtoolized ]; then
+		local AT_NOELIBTOOLIZE="yes"
+	fi
+	eautoreconf
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: Use `GNOME2_EAUTORECONF`?

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+src_install() {
+	gnome2_src_install
+
+	# bug #338213
+	# Uses webkit's JIT. Needs mmap('rwx') to generate code in runtime.
+	# MPROTECT policy violation. Will sit here until webkit will
+	# get optional JIT.
+	pax-mark m "${D%/}"/usr/bin/liferea
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: ED instead of D to make Prefix happy.

**Comment by nE0sIghT** [comment]
@mgorny 
Thanks for review. Pushed changes.

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME2_EAUTORECONF="yes"
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit autotools gnome2 pax-utils python-single-r1
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: Remove unused inherits.

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: Please state the version bumped to in the commit message (on summary line).

File: net-news/liferea/liferea-1.12.2.ebuild [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+src_install() {
+	gnome2_src_install
+
+	# bug #338213
+	# Uses webkit's JIT. Needs mmap('rwx') to generate code in runtime.
+	# MPROTECT policy violation. Will sit here until webkit will
+	# get optional JIT.
+	pax-mark m "${ED%/}"/usr/bin/liferea
+
+	einfo "If you want to enhance the functionality of this package,"
```
Comment (mgorny) [inline net-news/liferea/liferea-1.12.2.ebuild]: einfo is only for progress reporting. Also post-install messages belong in postinst.

**Comment by nE0sIghT** [comment]
All done.

File: net-news/liferea/liferea-1.12.2.ebuild (hunk 6 above)
Comment (mgorny) [reply net-news/liferea/liferea-1.12.2.ebuild]: Well, I meant after the version bump, not in package name. I'll fix it.

**Pull request closed without merging**